src/annotations/components/annotationForm/AnnotationTimeInput.tsx

```diff
@@ -43,7 +45,28 @@ export const AnnotationTimeInput: FC<Props> = (props: Props) => {
   )
 
   const isValidTimeFormat = (inputValue: string): boolean => {
-    return moment(inputValue, timeFormat, true).isValid()
+    const isValid = moment(inputValue, timeFormat, true).isValid()
+    props.onValidityCheck(isValid)
+    return isValid
+  }
+
+  const getInputValidationMessage = (): string => {
+    // if the parent is giving us a message to display; do it.
+    // else make our own.
+
+    if (props.invalidMessage) {
+      return props.invalidMessage
+    }
+    if (timeValue === null) {
+      return '' // it's not filled in yet...don't show an error (parent takes care of required fields)
+    }
+
+    if (!isValidTimeFormat(timeValue)) {
+      return `Format must be ${timeFormat}`
+    }
+
+    // it is valid
+    return null
```

Comment:
shouldn't this return `''` and not `null`?

Reply:
added

Reply:
it doesn't really matter; but I changed it.  also realized that the parent doesn't actually check for 'required' when the date is null, so fixed that and adjusted the tests too.